server/sonar-server/src/main/java/org/sonar/server/property/InternalProperties.java

```diff
@@ -32,6 +32,8 @@
    */
   String DEFAULT_ORGANIZATION = "organization.default";
 
+  String ORGANIZATION_ENABLED = "organization.enabled";
```

Comment:
No, this constant has been introduced in the first commit which creates the WS api/organizations/enable_feature